locales/de.json

```diff
@@ -33,5 +33,6 @@
   "Invalid JSON config file: %s": "Fehlerhafte JSON-Config Datei: %s",
   "Path to JSON config file": "Pfad zur JSON-Config Datei",
   "Show help": "Hilfe anzeigen",
-  "Show version number": "Version anzeigen"
+  "Show version number": "Version anzeigen",
+  "Did you mean %s?": "Meinten Sie %s?"
```

Comment:
hooray! wonder what a good way to request more translations would be? perhaps reach back out to some of the original folks ... might be worth eyeballing if we want any more new text in the application first though, get it done in one chunk.